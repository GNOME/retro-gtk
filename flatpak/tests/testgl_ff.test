[Retro Reftest]
Path=/testgl_ff
Core=/app/lib/libretro/testgl_ff_libretro.so

[Frame 0]
State=Refresh
