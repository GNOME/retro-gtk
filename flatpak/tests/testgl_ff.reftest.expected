/testgl_ff/Boot: [libretro-test]: Got size: 320 x 240.
HW Context could not be initialized, exiting...
OK
/testgl_ff/Options: OK
/testgl_ff/0/State None: OK
/testgl_ff/0/Run: [libretro-test]: Got size: 320 x 240.
